apps/camera/style/recording-timer.css

```diff
@@ -5,6 +5,7 @@
   left: 0;
   z-index: 1;
 
+  visibility: hidden;
```

Comment:
The example you gave only works one way... Meaning that you'll only be able to notice the element fading out, not fading in.  Is that the desired behavior?